site/docs/concepts/architecture/control-plane.md

```diff
@@ -8,75 +8,99 @@ sidebar_position: 3
 
 The control plane is responsible for configuring and managing the system. It consists of several key components working together to manage the AI Gateway configuration.
 
-![Control Plane Architecture](/img/control_plane.png)
+```mermaid
```

Comment:
I did't add any styling stuff so looks boring but should work for now at least

<img width="894" alt="image" src="https://github.com/user-attachments/assets/15363f48-2f66-4da4-9ee4-6dccf359eb9f" />